### ISSUE TITLE
iPhone can't recognize QR codes with logo

### ISSUE THREAD

**Issue by T-mac-one** [gruhn/vue-qrcode-reader#445]
Hello, iPhone can't recognize QR codes with logo.

I tried to use iPhone11 and iPhone15, and the QR code with logo failed to be recognized.

Please help to see how to fix it.

**Comment by T-mac-one** [comment]
![code](https://github.com/user-attachments/assets/060d3c0a-1ad5-4a4d-9eef-3450abb65cdc)
For example, this QR code can't be scanned.

**Comment by T-mac-one** [comment]
For example, this QR code can't be scanned.
![qrcode](https://github.com/user-attachments/assets/fbbf827d-a8cc-40d5-b68a-568d4b28ef12)

**Comment by gruhn** [comment]
On my end it works (iPhone SE 2020, iOS 17.6.1). Have used the demo page? https://gruhn.github.io/vue-qrcode-reader/demos/FullDemo.html


<img src="https://github.com/user-attachments/assets/043f0404-b455-4f89-98fe-418a3b1fffca" width="400px" />

**Comment by T-mac-one** [comment]
Thank you for your quick response!

My iPhone 11 still doesn't work.

Is it because the resolution is not high enough, or is there no white edge around the QR code?

**Comment by T-mac-one** [comment]
https://github.com/user-attachments/assets/e998a1c0-17c8-4909-8c47-279ac4ddf842

**Comment by gruhn** [comment]
> My iPhone 11 still doesn't work.

Hmm, that's very odd.

> Is it because the resolution is not high enough, 

Hmm, it should be fine. At least your iPhone 15 is newer then mine. So it should a have at least the same resolution, right?

> or is there no white edge around the QR code?

That's what I suspected first. But why does it work for me then? Let's try it anyway, here I added some white margin:

![362520414-fbbf827d-a8cc-40d5-b68a-568d4b28ef12](https://github.com/user-attachments/assets/6e4a2cb0-49a4-4134-8535-d89aba2dbe32)
